project.md

```diff
@@ -1,33 +1,29 @@
 
-[DELETE ME - ADD YOUR PROJECT MINI PITCH HERE]
-
-Example - This colour picker will revolutionaize the way colours are picked for the web.
+MeetMap is a social map service that lets you start and join friend groups and find fun places to meet with the least amount of travel from the group. Our software uses the Google Maps API to find all the members of the group (or a default address) and gives you selection of the top 5 most popular locations for teens in order of how close they are to the members of the group. Select a location and the map will show directions for each member of the group from their current location to the destination. No more racking your brain for where to meet and how everyone is going to get there. With an easily buildable MVP in 7 weeks, this project also allows for tons of expansion.  It is now easier than ever to get together with your friends and WE do all the work. 
 
 ## Project Description
 
-[DELETE ME - ADD YOUR PROJECT DESCRIPTION HERE]
-
-Example - The app will be web based and allow users to pick a colour with the help of a colour wheel.
+The MVP version of this web app will use the addresses of n members of a group and show the top 5 retults based on:
+  - Places teens like to meet (Fast food resturaunts, parks, malls, movies theaters, etc.) 
+  - The least amount of net travel in the group (Based on distance or time) 
+  - The least amount of difference between the travel of all group members (So we don't get johnny walking 30 seconds, and timmy having     to bus 90 minutes) 
+  
+For proof of concept, we could use static addresses, but once that is a success the first expansion would be Geolocating the group members and showing the top 5 results based on their current location. This idea has so much potential for expansion including a chat service for the group, an android and apple app, searching results for certain types of location, and much more.   
 
 ## Project Audience
 
-[DELETE ME - ADD YOUR PROJECT AUDIENCE HERE]
+The audience is for teens who are having a hard time getting together due to lack of trasportation, and ideas of where to meet. To confirm the audience, I asked 15 students if this is an issue they have and 9 replied yes, 5 sometimes, and 1 no. Though this is the audience in mind when designing the project, the application would not be limited to that. Other common audiences could be businesses who need to meet outside of the office, or even groups looking for a place to eat together. This would be specifically useful to Vancouver island where people are more spread out so optimizing people's travel times would provide a huge benefit.  
 
-Example - The audience would be frontend developers and designers looking to pick colours for their work.
 
 ## Project Inspiration
 
-[DELETE ME - ADD YOUR PROJECT INSPIRATION HERE]
-
-Example - This project idea was inspired by the following websites: list of website.
+Inspired by Google Maps, and will be using the Google Maps API. 
```

Comment:
Hey Matt!

This is a really cool idea! You did an awesome job at outlining a MVP.